### PR TITLE
New section and small edits

### PR DESCRIPTION
**Pull request by dschust-r**
I've added the new proposed section (Biochem basics) and started to write an introduction to proteins and some (non-MS) analysis techniques. I was not sure how detailed that section should be so I kept it a bit more concise.
Also edited some small things in the introduction file (numbering, new section) and in some minor things in the QC software table.
I renumbered the other sections.

Please provide a description of your changes:

Please check the following about your suggested changed:
- [x] The new text has one sentence per line
- [x] preprints that are cited are listed as an issue for discussion and peer review

### CODE REVIEW

**Comment by jessegmeyerlab** [comment]
Thanks Dina, I should be able to review this tomorrow - I'm waiting to merge Norbert's changes since this PR updates all the section numbers

File: content/03.biochemistry-basics.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
+## Biochemistry Basics {.page_break_before}
+
+### Proteins
+
+Proteins are large biomolecules or biopolymers made up of amino acids which are linked by peptide bonds. 
+They perform various functions in living organisms ranging from having structural roles to functional involvement in cellular signaling and the catalysis of chemical reactions (enzymes). 
+Proteins are made up of 20 different amino acids (not counting pyrrolysine and selenocysteine, which only occur in specific organisms) and their sequence is encoded in their corresponding genes.
+The human genome encodes more than 20,000 different proteins of varying abundance.
```
Comment (jessegmeyerlab) [inline content/03.biochemistry-basics.md]:
```suggestion
The human genome encodes more than 20,000 different proteins.
Each protein is present at a different abundances.
```

File: content/03.biochemistry-basics.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+Proteins are large biomolecules or biopolymers made up of amino acids which are linked by peptide bonds. 
+They perform various functions in living organisms ranging from having structural roles to functional involvement in cellular signaling and the catalysis of chemical reactions (enzymes). 
+Proteins are made up of 20 different amino acids (not counting pyrrolysine and selenocysteine, which only occur in specific organisms) and their sequence is encoded in their corresponding genes.
+The human genome encodes more than 20,000 different proteins of varying abundance. 
+Previous studies have shown that the concentration range of proteins can span over a range of at least seven orders of magnitude to up to 20 000 000 copies per cell and that their distribution is tissue-specific[@DOI:10.1038/msb.2011.82][@DOI:10.1016/j.cell.2020.08.036].
+Due to genetic variations, as well as alternative splicing and post-translational modifications, multiple different proteoforms can be produced from one single gene[@DOI:10.1038/nmeth.2369].
+
+#### PTMs
+
+Proteins can be post-translationally modified through enzymatic and non-enzymatic reactions _in vivo_ and _in vitro_.
```
Comment (jessegmeyerlab) [inline content/03.biochemistry-basics.md]:
```suggestion
Proteins can be post-translationally modified through enzymatic and non-enzymatic reactions _in vivo_ and _in vitro_ [@DOI:10.1093/database/baab012]. 
```

File: content/03.biochemistry-basics.md [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+Due to genetic variations, as well as alternative splicing and post-translational modifications, multiple different proteoforms can be produced from one single gene[@DOI:10.1038/nmeth.2369].
+
+#### PTMs
+
+Proteins can be post-translationally modified through enzymatic and non-enzymatic reactions _in vivo_ and _in vitro_. 
+Post-translational modifications (PTMs) can be reversible or non-reversible covalent modifications of amino acids.
+The most commonly studied and biologically relevant post-translational modifications include ubiquitinations (Lys, Cys, Ser, Thr, N-term), succinylations (Lys), methylations (Arg, Lys, His, Glu, Asn, Cys), disulfide bonds (Cys-Cys), oxidations (Met, Trp, His, Cys), phosphorylations (Ser, Thr, Tyr, His), acetylations (Lys, N-term), glycosylations (Arg, Asp, Cys, Ser, Thr, Tyr, Trp) and lipidations.
+Post-translational modification of a protein can alter its function, activity, structure, location and interactions. 
+Deregulation of PTMs is linked to cellular stress and diseases.[@DOI:10.1038/s41570-020-00223-8]
```
Comment (jessegmeyerlab) [inline content/03.biochemistry-basics.md]:
```suggestion
Deregulation of PTMs is linked to cellular stress and diseases[@DOI:10.1038/s41570-020-00223-8].
```

File: content/03.biochemistry-basics.md [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+
+#### Protein Structure
+
+Almost all proteins (except for intrinsically disordered proteins) fold into 3D structures either by themselves or assisted through chaperones[@DOI:10.1007/s12013-021-00970-5].
```
Comment (jessegmeyerlab) [inline content/03.biochemistry-basics.md]:
```suggestion
Almost all proteins (except for intrinsically disordered proteins[@DOI:10.1146/annurev-biochem-072711-164947]) fold into 3D structures either by themselves or assisted through chaperones[@DOI:10.1007/s12013-021-00970-5]. 
```

**Review by jessegmeyerlab — CHANGES_REQUESTED** [review]
Yet another awesome contribution Dina! This will really help those who are coming in without any biochem background. Requesting a few minor changes.

**Comment by dschust-r** [comment]
Hi Jesse, 

The suggested changes look good to me, I committed them. It seems like there are some merge conflicts (probably because I worked on files that were changed after I had posted my pull-request). Let me know if I should do anything or if you can resolve those conflicts.

**Comment by jessegmeyerlab** [comment]
> 

Is it possible to remove the renaming of section 04 from this and then I will do it manually after ?

**Pull request closed without merging**